### DISCUSSION TITLE
Verze aplikací a init script RUM

### DISCUSSION THREAD

**Discussion by lucienevludova-direct** [canarytrace/documentation#196] [Q&A] [answered]
Dobrý den, 

chtěla jsem se zeptat, máme ve firmě aplikaci Canarytrace ve verzi `canarytrace:4.22.0-devops-20221121083205-28`, dále pak jsme na základě #195 upravili verzi Listeneru na verzi `listener-agent:1.46` a RUM na verzi `rum:2.8.10`. Jsou tyto verze nejaktuálnější? Úpravy proběhly na základě Backlog and Releases v dokumentaci v2doc. Kde prosím doporučujete sledovat informace o nejaktuálnějších verzích? Přes webové stránky jsme se chtěli připojit k Discord komunitě, ale odkaz bohužel nefungoval - Neplatná pozvánka.
Tak, jak máme verze nyní nastavené a jsou nasazené, měla by aplikace mít veškeré podklady, které se týkají strany serveru? Verze výše zmiňované jsou vzájemně kompatibilní? URL obsahující `/health` vypisuje Canarytrace RUM is a ready!. Je možné nyní přistoupit k přidání init scriptu na straně klienta? Poté by se nám již měly zaznamenávat veškeré informace?

Velice Vám děkuji.
S pozdravem
Nevludová

**Comment by rdpanek** [comment, marked as answer]
Dobrý den, 
předně se omlouvám, Vaši zprávu jsem úplně přehlédnul.

Obecně se nyní nacházíme v přepisu dokumentace a hlavního produktu Canarytrace. Nyní Canarytrace obsahuje testovací framework WebDriverIO a v nové verzi již bude obsahovat PlayWright a nebude se jmenovat Canarytrace ale Synthetic.

Pro dokumentaci k WebdriverIO použijte prosím stávající dokumentaci: https://canarytrace.com/docs/ a pro ostatní produkty jako je RUM a Listener novou dokumentaci na dočasném umístění https://v2doc.canarytrace.com/docs/listener/releases a https://v2doc.canarytrace.com/docs/rum/releases

Jsou tam uvedené i nejaktuálnější verze pro Listener a RUM. Canarytrace se již upgradovat nebude, místo něj bude někdy v Listopadu první release Synthetic. O tom se dozvíte v e-mailu a v nové kategorii zde v diskuzi https://github.com/canarytrace/documentation/discussions/categories/releases dejte si prosím follow.

Zároveň v diskuzi https://github.com/canarytrace/documentation/discussions můžete nadhazovat i Vaše nápady k tématu kolem Canarytrace. 

Pak, nejnovější docker images pro listener-agent najdete zde https://quay.io/repository/canarytrace/listener-agent?tab=tags a pro RUM zde https://quay.io/repository/canarytrace/rum?tab=tags nic méně aktuální docker image je vždy uveden i na release stránce každého produktu.

Pak na této stránce https://github.com/canarytrace/documentation/discussions/93 v diskuzi najdete i rozcestník k boardům, kde uvidíte, co se chystá do dalšího release a i detaily některých vývojových kroků, jako jsou třeba i důvody. V těchto boardech se může objevit i Váš nápad nebo i implementace opravy chyby.

Pojďme k RUM, pokud `/health` vrací Canarytrace RUM is a ready!, tak je RUM Server ready a můžete implementovat do frontendů, tak jak píšete.

Díky za Vaše dotazy, ptejte se na cokoliv a ještě jednou se omlouvám za tak dlouhou reakci.
